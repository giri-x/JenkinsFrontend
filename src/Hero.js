import React from 'react'
import './Hero.css'
import handIcon from './Assests/hand_icon.png'
import arrow from './Assests/arrow.png'
import heroImage from './Assests/hero_image.png'


const Hero = () => {
  return (
    <div className="hero">
      <div className="hero-left">
        <h2>GET YOUR DREAM CARRER</h2>
        <div>
            <div className="hero-hand-icon">
                <p>Technology</p>
                <img src={handIcon} alt="" />
            </div>
            <p>made </p>
            <p>for everyone</p>
        </div>
        <div className="hero-latest-btn">
            <div>Apply for Job Now</div>
            <img src={arrow} alt="" />
        </div>

      </div>
      <div className="hero-right">
        <img src={heroImage} alt="" />

      </div>
    </div>
  )
}

export default Hero
