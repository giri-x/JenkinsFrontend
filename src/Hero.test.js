import React from 'react';
import { render, fireEvent, waitFor } from '@testing-library/react';
import Hero from './Hero';

describe('Hero component', () => {
  it('renders correctly', () => {
    const { container } = render(<Hero />);
    expect(container).toMatchSnapshot();
  });

  it('renders the correct text', () => {
    const { getByText } = render(<Hero />);
    expect(getByText('GET YOUR DREAM CARRER')).toBeInTheDocument();
    expect(getByText('Technology')).toBeInTheDocument();
    expect(getByText('made')).toBeInTheDocument();
    expect(getByText('for everyone')).toBeInTheDocument();
    expect(getByText('Apply for Job Now')).toBeInTheDocument();
  });



 
});