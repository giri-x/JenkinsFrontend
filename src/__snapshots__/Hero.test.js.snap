// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component renders correctly 1`] = `
<div>
  <div
    class="hero"
  >
    <div
      class="hero-left"
    >
      <h2>
        GET YOUR DREAM CARRER
      </h2>
      <div>
        <div
          class="hero-hand-icon"
        >
          <p>
            Technology
          </p>
          <img
            alt=""
            src="hand_icon.png"
          />
        </div>
        <p>
          made 
        </p>
        <p>
          for everyone
        </p>
      </div>
      <div
        class="hero-latest-btn"
      >
        <div>
          Apply for Job Now
        </div>
        <img
          alt=""
          src="arrow.png"
        />
      </div>
    </div>
    <div
      class="hero-right"
    >
      <img
        alt=""
        src="hero_image.png"
      />
    </div>
  </div>
</div>
`;
